File: StarterTemplate/Python/functions_activity3.py
def prime_number(number):
    prime_num = True
    if number <= 1:
        return "the definition says that numbers greater than one..."
    else:
        for i in range(2,number):
            if number % i == 0:
                prime_num = False
                return f"The number {number} is not a prime number."
    
    return f"The number {number} is a prime number."


number = int(input("Enter a natural number greater than 1 to check: "))
print(prime_number(number))

File: Java Script/functions_activity3.js
// function to check if a number is greater than 1;
function greaterThan1(number) {
  if (number > 1) {
    return true;
  } else return false;
}

// function checks if a number is prime
const primeNumbers = (number) => {
  //   if it is the number 2 it returns true
  if (number === 2) {
    return true;
  } else {
    // if it is greater than 2 then divide that number by all numbers between 2 and that number (excluding it)
    for (let i = 2; i < number; i++) {
      //   returns false if the number is evenly divisible
      if (number % i === 0) {
        return false;
      }
    }
    // if it didn't return false then return true
    return true;
  }
};

// sample number
const b = 63;
// if the greaterThan1 function is true, run the primeNumbers function and displays the appropriate information
if (greaterThan1(b)) {
  if (primeNumbers(b)) {
    console.log(`The number ${b} is a prime number.`);
  } else {
    console.log(`The number ${b} is not a prime number.`);
  }
} else {
  console.log("Only numbers greater than 1 may or may not be prime.");
}
